rosbag2_storage_default_plugins/test/rosbag2_storage_default_plugins/sqlite/test_sqlite_storage.cpp

```diff
@@ -126,76 +132,38 @@ TEST_F(StorageTestFixture, get_all_topics_and_types_returns_the_correct_vector)
   }));
 }
 
-TEST_F(StorageTestFixture, get_metadata_returns_correct_struct) {
```

Comment:
So if you decide to delete these tests, they have to be reintegrated in the rosbag2 API. I think it makes sense to modify this PR to also remove the `get_metadata()` function from the storage interfaces and handle the logic completely in the c++ API of rosbag2.

Reply:
`get_metadata` will be removed in Issue #197. Restored test for this PR.